templates/SpellC.html

```diff
@@ -0,0 +1,68 @@
+<html>
+<head>
+
+
+<style>
+
+body
+.navbar {
+  overflow: hidden;
+  background-color: #ddeeee;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+    textarea[type=text] {
+        width: 50%;
+        height:10%;
+        padding: 12px 20px;
+        margin: 8px 0;
+        box-sizing: border-box;
+        border: 2px solid black ;
+        border-radius: 4px;
+    }
+
+</style>
+
+
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+</head>
+<body>
+
+<nav class="navbar">
+  <div id="main">
+  <span style="font-size:30px;cursor:pointer"</span>
+<a class="navbar-brand"  href="#" style="color:rgb(226, 15, 15)" >
+    SpellChecker
+  </a>
+
+ </div>
+</nav>
+<br><br><br><br>
+
+            <div align="center">
+
+                <br />
+                <textarea type="text" id="myInput" placeholder="Initiate here..." name="lname">
+</textarea>
+                <br />
+		<form method="POST">
+    		    <input name="text">
+    		    <input type="submit">
+		</form>
+
+                <br />
+                <button type="button" class="btn btn-outline-dark">Enter</button>
+                
+                
+                <br />
+                <br />
+                <button type="button" onclick="document.getElementById('myInput').value = ''" class="btn btn-outline-dark">Reset</button>
```

Comment:
All the text which was entered in the text area will be cleared using this command.